docs/source/guides/statistical_insights.ipynb

```diff
@@ -4,11 +4,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "# Gain Statistical Insights into Your DataTable\n",
+    "# Gain Statistical Insights into Your Data\n",
     "\n",
-    "Woodwork provides methods on DataTable to allow you to use the typing information inherent in a DataTable to better understand your data.\n",
+    "Woodwork provides methods on your dataframes to allow you to use the typing information stored by Woodwork to better understand your data.\n",
     "\n",
-    "Follow along to learn how to use `describe` and `mutual_information` on a retail DataTable so that you can see the full capabilities of the functions."
+    "Follow along to learn how to use `describe` and `mutual_information` on a Woodwork initialized dataframe of retail data so that you can see the full capabilities of the functions."
```

Comment:
Should probably also include `value_counts` in this sentence

Reply:
How about if we make this more generic instead, so we won't have to remember to update all the time if we add more methods?

Reply:
that sounds good to me!

Reply:
Cleaned up the wording of this sentence and removed mentions of specific methods.